spec/Chassis/Chassis.vspec

```diff
@@ -43,6 +37,15 @@ AxleCount:
 #
 # Axle attributes
 #
+
+
+Axle.Track:
```

Comment:
Based on the discussion and definition in J1100 Revised NOV2009 I see that we could have at 3 signals

- Axle.TreadWidth (W101, The lateral distance between the centerlines of the base tires at ground, including camber angle. If there are dual wheels, measure from the midway points between the inner and outer tires)
- Axle.TrackWidth (W102, Track Width at Wheel Center. If there are dual wheels, measure from the midway points between the inner and outer tires)
- Axle.Width (W103, The lateral distance between the wheel mounting faces, measured along the spindle axis. Measurement is independent of both wheel offset and camber angle.)

As of now the PR only include one of them (W102). 

(We could also discuss if we would like an Axle.WheelOffset as well, i.e. ET = einpresstiefe)


I think having them in Axle branch is OK, if we consider the Axle branch to contain both information on the Axle itself, but also information concerning Axle including all wheels mounted to it.